src/main/resources/portal.properties.EXAMPLE

```diff
@@ -216,3 +216,4 @@ oncoprint.defaultview=patient
 # Disable cache (for non authorized portals this can sometimes avoid running
 # out of memory with relatively little performance loss)
 # mybatis.cache.enabled=false
+
```

Comment:
I missed this - do you want to remove this file from the changeset?

Reply:
It probably caused due to my previous implementation. Let me update.